flaky-tests/ci-providers/semaphore-ci-quickstart.md

```diff
@@ -24,6 +24,25 @@ In your Semaphore dashboard, store your Trunk token in a secret named TRUNK\_TOK
 
 ## Sample Semaphore workflow steps
 
+You can upload test results to Flaky Tests with
+the [`trunk-analytics-uploader`](https://github.com/trunk-io/analytics-uploader) by running it in a stage after
+your tests are complete. There are five different OS/arch builds of the uploader in the latest release.
+Pick the one you need for your testing platform and be sure to download the release on every CI run.
+Do not bake the CLI into a container or VM. This ensures your CI runs are always using the latest build.
+
+Right click and copy the appropriate link from this table.
+
+| CPU Architecture    | link                                                                                                                                                   |
+|---------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------|
+| MacOS Intel         | [x68_64-apple-darwin](https://github.com/trunk-io/analytics-cli/releases/latest/download/trunk-analytics-cli-x86_64-apple-darwin.tar.gz)               |
```

Comment:
<img width="536" alt="Screenshot 2024-08-09 at 11 55 45 AM" src="https://github.com/user-attachments/assets/2ed26799-d822-4637-883e-ae8c2d5df594">
nit: It's actually officially macOS not without capital. (no need to change, I think apple is wrong)

Reply:
ah, I thought apple had changed back to normal caps. Ha hah.

Reply:
changed.